src/Redux/Actions/readPopularActions.js

```diff
@@ -0,0 +1,17 @@
+import types from './index';
+import axios from '../../configs/axios';
+
+const { GET_POPULAR_ARTICLES } = types;
+
+export default {
+  readPopularArticle: () => (dispatch) => {
+    axios
+      .get('/api/articles/most/popular')
```

Comment:
@MCFrank16 what is this URL `/api/articles/most/popular` though? We've spoken about proper route naming before.